backend/src/test/java/com/bakdata/conquery/integration/tests/EntityExportTest.java

```diff
@@ -0,0 +1,95 @@
+
+import static com.bakdata.conquery.integration.common.LoadingUtil.importSecondaryIds;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import com.bakdata.conquery.integration.common.LoadingUtil;
+import com.bakdata.conquery.integration.json.JsonIntegrationTest;
+import com.bakdata.conquery.integration.json.QueryTest;
+import com.bakdata.conquery.models.auth.entities.Subject;
+import com.bakdata.conquery.models.common.Range;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.datasets.concepts.Concept;
+import com.bakdata.conquery.models.datasets.concepts.Connector;
+import com.bakdata.conquery.models.exceptions.ValidatorHelper;
+import com.bakdata.conquery.resources.ResourceConstants;
+import com.bakdata.conquery.resources.api.QueryResource;
+import com.bakdata.conquery.resources.hierarchies.HierarchyHelper;
+import com.bakdata.conquery.util.support.StandaloneSupport;
+import com.bakdata.conquery.util.support.TestConquery;
+import com.github.powerlibraries.io.In;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.description.LazyTextDescription;
+
+/**
+ * Adapted from {@link com.bakdata.conquery.integration.tests.deletion.ImportDeletionTest}, tests {@link com.bakdata.conquery.resources.api.QueryResource#getEntityData(Subject, QueryResource.EntityPreview, Optional, HttpServletRequest)}.
+ */
+@Slf4j
+public class EntityExportTest implements ProgrammaticIntegrationTest {
+
+
+	@Override
+	public void execute(String name, TestConquery testConquery) throws Exception {
+
+		final StandaloneSupport conquery = testConquery.getSupport(name);
+
+		final String testJson = In.resource("/tests/query/DELETE_IMPORT_TESTS/SIMPLE_TREECONCEPT_Query.test.json").withUTF8().readAll();
+
+		final Dataset dataset = conquery.getDataset();
+
+		final QueryTest test = (QueryTest) JsonIntegrationTest.readJson(dataset, testJson);
+
+		// Manually import data, so we can do our own work.
+		{
+			ValidatorHelper.failOnError(log, conquery.getValidator().validate(test));
+
+			importSecondaryIds(conquery, test.getContent().getSecondaryIds());
+			conquery.waitUntilWorkDone();
+
+			LoadingUtil.importTables(conquery, test.getContent().getTables());
+			conquery.waitUntilWorkDone();
+
+			LoadingUtil.importConcepts(conquery, test.getRawConcepts());
+			conquery.waitUntilWorkDone();
+
+			LoadingUtil.importTableContents(conquery, test.getContent().getTables());
+			conquery.waitUntilWorkDone();
+		}
+
+		final URI entityExport = HierarchyHelper.hierarchicalPath(conquery.defaultApiURIBuilder(), QueryResource.class, "getEntityData")
+												.buildFromMap(Map.of(ResourceConstants.DATASET, conquery.getDataset().getName()));
```

Comment:
Are indentations intended to be all over the place like this?

Reply:
It looks garbled online, but i've set up my IDE to align on dots:

![image](https://user-images.githubusercontent.com/1553491/168572094-7b646dd0-c740-4feb-a1fa-ead6b62f14eb.png)